acai-ios/MenuViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  MenuViewController.swift
+//  acai-ios
+//
+//  Created by Jaewon Sim on 3/13/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import IGListKit
+import SnapKit
+
+class MenuViewController: UIViewController {
+    
+    var collectionView: UICollectionView!
+    var adapter: ListAdapter!
+
+    // MARK: Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view.backgroundColor = .white
+        
+        collectionView = UICollectionView(frame: .zero, collectionViewLayout: UICollectionViewFlowLayout())
+        view.addSubview(collectionView)
+        
+        adapter = ListAdapter(updater: ListAdapterUpdater(), viewController: self)
+        adapter.collectionView = collectionView
+        adapter.dataSource = self
+        
+        setUpConstraints()
+    }
+    
+    // MARK: Constraint setup
+    func setUpConstraints() {
```

Comment:
should be `setupConstraints`, should this code just be in the `init` if it's so short? @kevinchan159